docs/specs/sequences/endpoint_reconfiguration.md

```diff
@@ -57,6 +57,21 @@ sequenceDiagram
 
 ```
 
+### Commit Reconfiguration (Successful reconfiguration, persists Reboot)
+- Documentation on [device reboot mechanism through UDMI](../system_mode.md)
+
+```mermaid
+%%{wrap}%%
+sequenceDiagram
+    autonumber
+    participant D as Device
+    participant E as Original Endpoint
+    participant E' as New Endpoint
+    D->>E':STATE MESSAGE<br/>blobset.blobs._iot_endpoint_config.phase = "final"
+    note over D,E': Reboot sequence from new endpoint
```

Comment:
nit: I'm calling it "restart" not "reboot" not sure what the exact difference would be -- but I was going with the terminology my laptop uses :-).